release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -85,6 +85,28 @@ CRI-O brings:
 
 image::crio-3-7.png[CRI-O]
 
+To install and run CRI-O alongside `docker`, set the following in the
+`[OSEv3:vars]` section
+xref:../install_config/install/advanced_install.adoc#configuring-ansible[Ansible inventory file] during cluster installation:
+
+----
+openshift_use_crio=True
+----
+
+This setting pulls the *openshift3/cri-o:latest* image from the link:https://access.redhat.com/containers[Red Hat Registry]. It also sets
+`openshift_docker_systemcontainer_image_registry_override` as its override just
```

Comment:
Did you mean ``openshift_crio_systemcontainer_image_override``?

Reply:
It mentions `openshift_docker_systemcontainer_image_registry_override` in https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.example#L118.

Reply:
I had also mention to talk about `openshift_crio_systemcontainer_image_override`, so I've now replaced the stuff from https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.example#L118 with talk about overriding the default image instead. If the line from https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.example#L118 can be ignored, I'll just leave it out.